satpy/writers/__init__.py

```diff
@@ -325,17 +327,44 @@ def get_filename(self, **kwargs):
                 "No filename pattern or specific filename provided")
         return self.filename_parser.compose(kwargs)
 
-    def save_datasets(self, datasets, **kwargs):
+    def save_datasets(self, datasets, compute=True, **kwargs):
```

Comment:
The docstring should be updated to exlain `compute`, and how that affects the return value.